.github/workflows/ci-build-java8.yml

```diff
@@ -0,0 +1,33 @@
+name: CI Build with Java 8
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  java-8-compatibility:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: 8
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: Run maven compile with Java 8
+        run: |
+          mvn --batch-mode --update-snapshots clean compile \
```

Comment:
Unfortunately, I cannot do it, since it tries to compile `exasol-testcontainers` which was compiled by Java 11
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.10.1:testCompile (default-testCompile) on project glue-connector: Compilation failure
[ERROR] /home/mtoraz/Devel/git/work/master/glue-connector/src/test/java/com/exasol/glue/ittests/BaseIntegrationTest.java:[11,29] cannot access com.exasol.containers.ExasolContainer
[ERROR]   bad class file: /home/mtoraz/.m2/repository/com/exasol/exasol-testcontainers/6.1.1/exasol-testcontainers-6.1.1.jar(com/exasol/containers/ExasolContainer.class)
[ERROR]     class file has wrong version 55.0, should be 52.0
[ERROR]     Please remove or make sure it appears in the correct subdirectory of the classpath.
```